index.html

```diff
@@ -0,0 +1,258 @@
+        </div>
+      </div>
+    </section>
+    <!-- About Section -->
+    <section class="about-page" id="about-page">
+      <div class="about-container">
+        <div class="about-content">
+          <h2 class="about-title">About Myself</h2>
+          <p class="about-para">
+            Hello I'm a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don't
+            hestiate to contact me.
+          </p>
+          <h3 class="about-contact">LET'S CONNECT</h3>
+          <ul class="social-icons">
+            <li class="icons">
+              <a href="#"><img src="images/Vector (15).png" alt="Twitter" /></a>
+            </li>
+            <li class="icons">
+              <a href="#"
+                ><img src="images/Linkedin icon.png" alt="LinkedIn"
+              /></a>
+            </li>
+            <li class="icons">
+              <a href="#"><img src="images/Vector (11).png" alt="Medium" /></a>
+            </li>
+            <li class="icons">
+              <a href="#"><img src="images/Vector (12).png" alt="GitHub" /></a>
+            </li>
+            <li class="icons">
+              <a href="#"
+                ><img src="images/Vector (13).png" alt="AngelList"
+              /></a>
+            </li>
+          </ul>
+          <button class="about-btn">Get my resume</button>
+        </div>
+        <div class="about-lang">
+          <div class="languages">
+            <div class="lang-title">
+              <h3>Languages</h3>
+              <img  src="images/Union.svg" alt="arrow" />
+            </div>
+
+            <ul class="lang">
+              <li class="langs">
+                <img
+                 src="images/Ellipse 1.png"
+                  alt="JavaScript-Logo"
+                />
+                <h4 class="langs-text">JavaSript</h4>
+              </li>
+              <li class="langs">
+                <img src="images/Ellipse 2.png" alt="HTML5-Logo" />
+                <h4 class="langs-text">HTML</h4>
+              </li>
+              <li class="langs">
+                <img src="images/Ellipse 3.png" alt="CSS3-Logo" />
+                <h4 class="langs-text">CSS</h4>
+              </li>
+            </ul>
+          </div>
+          <div>
+            <ul>
+            <div class="framework">
+             
+             <li><h4 class="frame-title">Frameworks</h4></li>
+           <li><img src="images/Union2.svg" alt="arrow" /></li>   
+            </div>
+            <hr class="new1" />
+          </div>
+          <div>
+            <div class="skill">
+             <li><h4 class="skill-title">Skills</h4></li> 
+             <li><img  src="images/Union2.svg" alt="arrow" /></li>
+            </div>
+           <hr class="new1" />
+          </div>
+        </ul>
+        </div>
+    
+    </section>
```

Comment:
- [ ] It is required to stick to the Figma design as close as possible. This involves font, color, padding, margin among others. Your design does not look the same as the Figma design. Kindly give a closer look to the Figma file and work your design alongside the Figma design. Take a look :point_down: 

### Your Design
![your-about-section](https://user-images.githubusercontent.com/26597432/235945485-280d7409-9f6a-431e-9e3d-693a52f4e2f1.png)

### Figma Design

![temp-about-1](https://user-images.githubusercontent.com/26597432/235945606-1e201d79-7176-4dab-903c-3a034b8f7e4a.png)

![temp-about-2](https://user-images.githubusercontent.com/26597432/235945660-3a96d987-3ef6-4355-85bd-2bf847b521d4.png)